### PR TITLE
[team]: Add docker-team dependencies

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: rules/docker-team.mk [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 DOCKER_TEAM = docker-team.gz
 $(DOCKER_TEAM)_PATH = $(DOCKERS_PATH)/docker-team
-$(DOCKER_TEAM)_DEPENDS += $(SWSS)
+$(DOCKER_TEAM)_DEPENDS += $(SWSS) $(LIBTEAM) $(LIBTEAMDCT) $(LIBTEAM_UTILS)
```
Comment (marian-pritsak) [inline rules/docker-team.mk]: swss already has a runtime dependency on team
https://github.com/Azure/sonic-buildimage/blob/master/rules/swss.mk#L7
This is redundant

Comment (stcheng) [inline rules/docker-team.mk]: updated to remove redundant dependencies

File: rules/swss.mk [hunk 2]
```diff
@@ -4,6 +4,5 @@ SWSS = swss_1.0.0_amd64.deb
 $(SWSS)_SRC_PATH = $(SRC_PATH)/sonic-swss
 $(SWSS)_DEPENDS += $(LIBSAIREDIS_DEV) $(LIBSAIMETADATA_DEV) $(LIBTEAM_DEV) \
     $(LIBTEAMDCT) $(LIBTEAM_UTILS)
-$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM) $(LIBTEAMDCT) \
-    $(LIBTEAM_UTILS)
+$(SWSS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBTEAM)
```
Comment (marian-pritsak) [inline rules/swss.mk]: Just to clarify: so those two are only needed for for docker-team?

Comment (stcheng) [inline rules/swss.mk]: yes. teamd/teamdctl commands are only supposed to work in docker-team

**Pull request merged**